src/main/java/org/apache/commons/lang3/BitField.java

```diff
@@ -56,15 +56,15 @@
  * <p>Flags and data can be retrieved from the integer:</p>
  *
  *<pre>
- *    // Prints true if red, green or blue is non-zero
+ *    // Prints {@code true} if red, green or blue is non-zero
```

Comment:
This is a code comment, not a Javadoc comment, so using Javadoc decorations is wrong. Please review all the files and revert this type of change please.

Reply:
done. Looks like only was this mistake.

Reply:
@arturobernalg 
You need to update the PR...

Reply:
ups. Sorry @garydgregory 
it's done.